metrics/default_metrics/tarantool/fibers.lua

```diff
@@ -0,0 +1,26 @@
+local fiber = require('fiber')
+local utils = require('metrics.default_metrics.tarantool.utils');
+
+local function get_fibers_metrics()
```

Comment:
Maybe update_fibers_metrics instead of get?

There is confusion with current naming as it seems the function returns memory_metrics. But it just updates / creates some gauges.